package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hospital-managment-software",
+  "version": "1.0.0",
+  "description": "Mange patients record across hospital personnel",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run build && babel-node src/server.js",
+    "start:dev": "nodemon --exec babel-node src/server.js",
+    "clean": "rm -rf build && mkdir build",
+    "build": "npm run clean && npm run build-babel",
+    "build-babel": "babel -d ./build ./src -s",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Maxfurry/Hospital-Managment-Software.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Maxfurry/Hospital-Managment-Software/issues"
+  },
+  "homepage": "https://github.com/Maxfurry/Hospital-Managment-Software#readme",
+  "dependencies": {
+    "@babel/core": "^7.12.3",
+    "@babel/node": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.6"
```

Comment:
@oluwatobiloba2000 Please move, @babel/core, @babel/node, @babel/preset-env and nodemon to devDepency